src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs

```diff
@@ -1137,13 +1138,13 @@ public static List<FontInfo> GetFontInfoList(PropertyArray fontArray)
         private const float FontSizeScaleGiant = 2.5f;
 #else   // PROFILE_MOBILE and etc
         // The following values from 'system-settings/libutil/sstu.c'
-        private const float FontSizeScaleSmall = 0.8f;
+        private const float FontSizeScaleSmall = 0.87f;
         private const float FontSizeScaleNormal = 1.0f;
         // TODO: Profile Separation
-        private const float FontSizeScaleLarge = 1.0f;
+        private const float FontSizeScaleLarge = 1.1f;
         //private const float FontSizeScaleLarge = 1.5f;
-        private const float FontSizeScaleHuge = 1.9f;
-        private const float FontSizeScaleGiant = 2.5f;
+        private const float FontSizeScaleHuge = 1.12f;
```

Comment:
This should be 1.2

Reply:
Oh my.......  :(
Thank you for checking the value! Fixed it!